spec/nameable_spec.rb

```diff
@@ -0,0 +1,8 @@
+# require_relative 'spec-helper'
+
+# describe Nameable do
+#   it "Should raise a NotImplemented error" do
+#     name = Nameable.new
+#     expect(name.correct_name()).to raise_error(NotImplementedError)
+#   end
+# end
```

Comment:
### Optional
- [ ] It would be better if we remove all commented piece of code, this makes our codebase looks messy and difficult to understand

Reply:
Don e